sessionexp.js

```diff
@@ -31,12 +30,8 @@ const parseJwt = (token) => {
     let time_left_min = get_mins((expd - curr)/1000)
     document.title = time_left_min;
     $("#cb-timer").html(`Time left: ${time_left_min}`);
-    if(expd - curr < 10000){
-      try{
-        (new Audio(mp3_url)).play();
-      } catch (e) {
-        alert("session is expiring");
-      }
+    if(Math.abs(expd - curr) < 10000){
+      aud.play().catch(e=>{alert("Session expired! Please logout and login again")});
```

Comment:
Set a count for how many times this will play, otherwise if a user remains on dashboard, it will continue to play infinitely.
For the alert you could make the count 1 and for sound make it 4 or 5

Reply:
Hey it will play 2-4 times. Which I think is decent.

Reply:
Where is the 2 - 4 times logic?

Reply:
Okay, so basically +10000 to -10000 which is 20 secs in intervals of 5